src/test/java/com/google/api/codegen/testdata/java_test_library.baseline

```diff
@@ -637,7 +631,7 @@ public class LibraryServiceTest {
     mockLibraryService.setResponses(expectedResponses);
 
     String formattedName = LibraryServiceApi.formatBookName("[SHELF]", "[BOOK]");
-    String indexName = "indexName746962392";
+    String indexName = "default index";
```

Comment:
We want to distinguished default values for samples from auto-generated values for tests. For example, here we would prefer an randomized unique index name instead of a default index.